controllers/remote/cluster_cache.go

```diff
@@ -68,7 +68,7 @@ type ClusterCacheTracker struct {
 type ClusterCacheTrackerOptions struct {
 	// Log is the logger used throughout the lifecycle of caches.
 	// Defaults to a no-op logger if it's not set.
-	Log logr.Logger
+	Log *logr.Logger
```

Comment:
Note: logr.Logger is now a struct not an interface. I made it a pointer to be able to check if the option is set.

I think there is simply no way to keep the API stable when go-logr changes the Logger interface to a struct (from v0.4=>v1.x)